client/src/components/viewers/FileControlsInput.vue

```diff
@@ -0,0 +1,146 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <span
+    v-if="!editing"
+    class="file-controls-input text-only"
+    @click="onTextClick"
+  >
+    {{ modelValue }}
+    {{ props.suffix }}
+  </span>
+  <ion-input
+    v-else
+    class="file-controls-input editing-input"
+    fill="outline"
+    ref="inputRef"
+    :value="modelValue"
+    @ion-blur="onFocusChanged(false)"
+    @ion-focus="onFocusChanged(true)"
+    @ion-input="onChange($event.detail.value || '')"
+    @keyup.enter="enterPressed($event.target.value)"
+    @keyup.esc="onFocusChanged(false)"
+    :disabled="$props.disabled"
+    v-bind="$attrs"
```

Comment:
I would avoid the `v-bind` here, it could really mess up the behavior if the wrong kind of props are passed.

Reply:
done